examples/move/scratch_off/sources/game.move

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module scratch_off::game {
+
+    use std::vector;
+    use sui::coin::{Self, Coin};
+    use sui::balance::{Self, Balance};
+    use sui::object::{Self, UID, ID};
+    use sui::tx_context::{Self, TxContext};
+    use sui::dynamic_object_field as dof;
```

Comment:
Do you need to use a dynamic object field (instead of dynamic field?)

Reply:
No specific feeling about this one. It can fit in either